### PR TITLE
update the main readme

### PR DESCRIPTION
**Pull request by klavs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,32 +8,33 @@ It consists of multiple packages and libraries centered around GraphQL AST.
...
 #### Code generation
 
-Current code generator is capable of building a Dart representation of GraphQL documents. It can be used to pre-build AST of `*.graphql` documents.
+GraphQL language enables code generation to speed up development and move document processing to build-time.
+  
+We provide code builders to generate Dart source code and file builders which easily plug into standard Dart code generation pipeline.
 
 Ideas for future work:
 - Dart types from a GraphQL schema
 - Dart types per GraphQL operation and it's variables (see [Artemis project][artemis])
```
Comment (smkhalsa) [inline README.md]: Should this be removed from "Ideas for future work"?

File: README.md [hunk 2]
```diff
@@ -76,19 +90,36 @@ Ideas for future work:
...
 |---------|-------------|
 | [artemis][artemis] | Build dart types from GraphQL schemas and queries (using Introspection Query). | 
-| [graphql fork][graphql_flutter] | A stand-alone GraphQL client for Dart, bringing all the features from a modern GraphQL client to one easy to use package. |
-| [graphql_flutter fork][graphql_flutter] | A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package. |
+| [graphql][graphql_flutter] | A stand-alone GraphQL client for Dart, bringing all the features from a modern GraphQL client to one easy to use package. |
+| [graphql_flutter][graphql_flutter] | A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package. |
+| [normalize][normalize] | Normalization and denormalization of GraphQL responses in Dart |
+| [gql_client][gql_client] | GraphQL Client for Dart |
+| [graphql-to-dart][graphql-to-dart]| generate dart classes and respective JsonSerializable transcoders |
+| [graphql_server][graphql_server]| Base package for implementing GraphQL servers. |
```
Comment (klavs) [inline README.md]: @thosakwe, is this the only package currently directly using any of the packages in this repo?

File: README.md (hunk 1 above)
Comment (klavs) [reply README.md]: 🙇

File: README.md (hunk 2 above)
Comment (thosakwe) [reply README.md]: Technically, `angel_graphql` uses `gql`, BUT it's indirect - it only depends on `graphql_server` directly. So, I would say yes, `graphql_server` is the only one.

Comment (klavs) [reply README.md]: Thanks for clarification!

**Pull request merged**